public/app/features/alerting/unified/components/rules/RuleActionsButtons.tsx

```diff
@@ -187,6 +183,22 @@ export const RuleActionsButtons: FC<Props> = ({ rule, rulesSource }) => {
             onDismiss={() => setRuleToDelete(undefined)}
           />
         )}
+        <ConfirmModal
+          isOpen={!!provRuleCloneUrl}
+          title="Clone provisioned rule?"
```

Comment:
I don't think we need the `?` here as we are just letting them know if the intention